yt_dlp/extractor/manyvids.py

```diff
@@ -130,33 +109,22 @@ def mung_title(s):
             if f.get('height') is None:
                 f['height'] = int_or_none(
                     self._search_regex(r'_(\d{2,3}[02468])_', f['url'], 'video height', default=None))
-            if '/preview/' in f['url']:
-                f['format_id'] = '_'.join(filter(None, (f.get('format_id'), 'preview')))
+            if 'preview' in f['format_id']:
                 f['preference'] = -10
             if 'transcoded' in f['format_id']:
                 f['preference'] = f.get('preference', -1) - 1
```

Comment:
All the formats code should be moved together. I don't like most of this old code, either. We should only set `preference` for the `teaser` format, and use `quality` to prioritize `filepath` over `transcoded`